scraper/src/github-scraper/parseEvents.ts

```diff
@@ -9,7 +9,19 @@ import { parseISO } from "date-fns";
 import { isBlacklisted } from "./utils.js";
 import { octokit } from "./config.js";
 const processedData: ProcessData = {};
-
+async function fetchdefaultbranch(org: string, repo: string) {
```

Comment:
ensure proper naming conventions.

```suggestion
async function getDefaultBranch(org: string, repo: string) {
```